app/views/kudos/index.html.erb

```diff
@@ -29,4 +29,5 @@
 
 <br>
 
-<%= link_to 'New Kudo', new_kudo_path, class:"btn btn-secondary" %>
+
+<%#= link_to 'New Kudo', new_kudo_path, class:"btn btn-secondary" %>
```

Comment:
Maybe get rid of that line instead of commenting it out?

Reply:
@ciejj imo your comment is a little passive-aggressive :P. I would write that leaving a commented-out code isn't a good practice

Reply:
Apologies - that was not my intention - I am still learning to comment code 😄